dataflow/gpu-workers/Dockerfile

```diff
@@ -0,0 +1,33 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM tensorflow/tensorflow:2.4.0-gpu
```

Comment:
We may want to make it an ARG with a default value to give some flexibility to users which base image they'd like to choose without editing the file. WDYT?

Reply:
In this case I wouldn't do that. The reason is that this Dockerfile is tailored specifically for that base image since that's the TensorFlow version defined in the requirements.txt file, and we're skipping the TensorFlow installation in it. If users use a different base image, it probably won't work and they'd probably need to modify the Dockerfile anyways. Also adding an ARG both makes the Dockerfile slightly less readable, and we would need to provide instructions on how to modify it. I would favor simplicity for the sample.